src/logger/index.js

```diff
@@ -47,15 +60,15 @@ const prodLoggingFormat = printf( output => {
 	return `${ time } ${ app }:${ type } ${ JSON.stringify( output ) }`;
 } );
 
-module.exports = ( namespace, { transport } = { } ) => {
+module.exports = ( namespace, { transport, cluster } ) => {
```

Comment:
I wonder, if we shouldn't start accepting an options object as the first parameter. I feel like there are probably N other things we could pass in, and the longer we go with just adding parameters, the harder it will be to change the API for this later.

Reply:
Alternate suggestions?

Reply:
Oh now that I look at this again, I see that it _is_ accepting an object of properties as the second arg - I initially read it as individual args before coffee :)

So yeah LGTM as-is!